packages/server/src/api/controllers/layout.ts

```diff
@@ -30,12 +30,12 @@ export async function destroy(ctx: BBContext) {
     layoutRev = ctx.params.layoutRev
 
   const layoutsUsedByScreens = (
-    await db.allDocs(
+    await db.allDocs<any>(
```

Comment:
I wasn't sure exactly what type this should be.

Reply:
This should be a `Layout` - but they are deprecated objects, not used in the platform anymore.